AuthenticationServiceTest/Helpers/HasherTest.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+using AuthenticationService.Helpers;
+using Xunit.Sdk;
+
+namespace AuthenticationServiceTest.Hasher
+{
+    public class HasherTest
```

Comment:
Zorg ervoor dat je een naming scheme aanhoud, zoals TestedMethod_Input_Return of VerifyHash_InvalidPassword_ThrowInvalidPasswordException/VerifyHash_InvalidSalt_ThrowInvalidSaltException

Reply:
Dit zorgt voor sneller kunnen zien wat er wordt getest in een lijstje etcetera en welke parameters er worden gezet met bijbehorende results.